### ISSUE TITLE
NCLscan and unpaired read data?

### ISSUE THREAD

**Issue by wolfgangrumpf** [TreesLab/NCLscan#20]
Is there a way to run NCLscan on non-paired-read data (e.g. PacBio output)?

**Comment by chenchiaying** [comment]
Hi,
 
NCLscan cannot apply to long reads. 

But, our team now is developing another NCL tool on long-reads (PacBio/Nanopore).  This new tool still needs a non-co-linear result (fusion, trans-splicing, and circular RNAs) from short-reads and looks for full-length of a non-co-linear event using long-reads. 

Chia-Ying

**Comment by wolfgangrumpf** [comment]
That's great news - do you need testers?  When do you expect the tool to be ready?

**Comment by chenchiaying** [comment]
Yes, we do need testers. Thanks for your generous help. The longNCL tool is planning to release in GitHub within 2 weeks. I will let you know when it is ready. Keep in touch:)

Chia-Ying
